.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,25 @@
+name: Coverage
+on:
+  - pull_request
```

Comment:
We don't want coverage running on `main`, only on PRs?

Reply:
Coverage report is only valuable on pull requests.